openid-4-verifiable-presentations-1_0.md

```diff
@@ -581,12 +581,24 @@ The following is a non-normative example of the Authorization Response that is s
     state=eyJhb...6-sVA
 ```
 
-If the Response Endpoint has successfully processed the request, it MUST respond with HTTPS status code 200. 
+The following is a non-normative example of an Authorization Error Response that is sent via an HTTPS request to the Verifier's Response Endpoint:
```

Comment:
```suggestion
The following is a non-normative example of an Authorization Error Response that is sent as an HTTP POST request to the Verifier's Response Endpoint:
```

Reply:
weird question, but why is authorization response sent via POST is form-urlencoded and not applicaiton/json..?
I know the direct_post definition says: `The Authorization Response parameters are encoded in the body using the application/x-www-form-urlencoded content type.` but..

Reply:
That is a separate issue. I would have preferred JSON as well. I don't think we can fix that without normative changes.

Reply:
For the record I think the reason is likely that it matches the format received at the redirect_uri (so being consistent makes it slightly easier for existing implementors), and that generally most/all OAuth2 endpoints expect application/x-www-form-urlencoded input.